src/tests/ftest/util/test_utils_pool.py

```diff
@@ -410,7 +410,9 @@ def get_info(self):
         Sets the self.info attribute.
         """
         if self.pool:
+            self.log.info("Connecting to pool %s", self.identifier)
```

Comment:
I find this message could be misleading as we may not actually connect to the pool if `self.connected` is True.  Also we already print a message if we do end up connecting to the pool: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_pool.py#L291-L293
```suggestion
```

Reply:
Oops, you're right!